canvas/fizz-27-py/src/lib/canvasRequest.py

```diff
@@ -0,0 +1,10 @@
+import requests
+import os
+
+def get(url):
+    base_url = os.getenv("CANVAS_BASE_URL")
+    auth_token = os.getenv("ACCESS_TOKEN")
```

Comment:
Eventually, if we create our own library interface, we would want to pass these values into a constructor instead of pulling them from the environment variables here.